File: Data Structures & Algorithms/missing-number/submission-0.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        
      int n=nums.size();
      int acts=n*(n+1)/2;
      int prs=0;
      for(int i=0;i<n;i++){
        prs+=nums[i];
       


      }

      if(acts==prs) return 0;

      else{
        return acts-prs;
      }
 



    }
};

File: Data Structures & Algorithms/number-of-good-pairs/submission-0.java
class Solution {
    public int numIdenticalPairs(int[] nums) {

        int n=nums.length;
        Map<Integer,Integer>mpp=new HashMap<>();
        for(int i=0;i<n;i++){
           
           mpp.put(nums[i],mpp.getOrDefault(nums[i],0)+1);

           
        } 
        int ans=0;

        for(Map.Entry<Integer,Integer>e:mpp.entrySet()){
            
            ans+=e.getValue()*(e.getValue()-1)/2;
            
        }
        return ans;
    }
}